### DISCUSSION TITLE
Work flow graph not displaying in admin console.

### DISCUSSION THREAD

**Discussion by AnoopIssacg10x** [pimcore/pimcore#17225] [Q&A / Support]
We installed PIMCORE community edition version 11.2.6 in windows. We configured a sample work flow for a data object but Work flow graph is not displaying in admin console.

![image](https://github.com/pimcore/pimcore/assets/173474664/e7cfbe35-b4ef-4125-93f9-c1c9f68682cc)

We installed the Graphviz version 11.0.0 and configured the path variable, but no result

When we run "dot" command , nothing coming , the cursor waits indefinitely

![image](https://github.com/pimcore/pimcore/assets/173474664/1fa273a2-259e-456b-849e-c939c2525129)

appreciate for any help

**Comment by markus-moser** [comment]
It seems that you are running Pimcore on a windows server. The detection of the "dot" command location does not work. Please set the parameter `pimcore_executable_dot` in your symfony config to the location of the dot binary.

So something similar to this (e.g. in your config.yaml):

```yaml
parameters:
   pimcore_executable_dot: 'c:\Program Files\Graphviz\bin\dot.exe'
```

**Reply by AnoopIssacg10x** [reply]
Thanks for the reply,

I did the same, but no result 

<img width="568" alt="image" src="https://github.com/pimcore/pimcore/assets/173474664/e7393e79-2659-42b5-8b84-fc390f6f8a2e">

<img width="601" alt="image" src="https://github.com/pimcore/pimcore/assets/173474664/879a3be0-2bfd-4d0f-9b18-36a4d6c1df1e">

It is strange that the  "dot" command is not working even I change directory to the D:\PimCore\Graphviz1100win64\bin folder

![image](https://github.com/pimcore/pimcore/assets/173474664/3bdf0072-9c5d-4d71-a4e2-c4014ce5898d)

seems like it's not the path issue, right?

**Reply by markus-moser** [reply]
Yes now it's no path issue anymore. The `dot` command is used here:

https://github.com/pimcore/admin-ui-classic-bundle/blob/0e14ad32b3b97a68c4feddc664589f292fa36194/src/Controller/Admin/WorkflowController.php#L348

You could try to execute the command directly in your command line and see what happens.

**Reply by AnoopIssacg10x** [reply]
Sorry, I didn't get get what you meant. I am new to pimcore and php too.
Did you mean run "dot" command from "D:\PimCore\projects\tj-pp-demo\vendor\pimcore\admin-ui-classic-bundle\src\Controller\Admin"
where "WorkflowController.php" placed. If so the result is same , the cursor waits indefinitely after executing dot command

**Reply by markus-moser** [reply]
I think the topic is that the command which is linked cannot work on windows because it useses a pipe "|" to combine the workflow console command with the dot command.

Is there a specific reason why you are running Pimcore on windows? You should really use linux for it. If you need it for local development use docker:
https://github.com/pimcore/skeleton?tab=readme-ov-file#docker
